alchemist-web-renderer/build.gradle.kts

```diff
@@ -95,6 +95,9 @@ application {
  * Webpack task that generates the JS artifacts.
  */
 val webpackTask = tasks.named("jsBrowserProductionWebpack")
+dependencies {
+    implementation(project(mapOf("path" to ":")))
+}
```

Comment:
What is this for?

Reply:
I don't recall adding it, it could be something IDE inserted. I see no use for it so I removed it.